coordinator/src/db/polls.rs

```diff
@@ -44,6 +49,7 @@ pub struct Poll {
     pub question: String,
     pub active: bool,
     pub creation_timestamp: OffsetDateTime,
+    pub whitelisted: bool,
```

Comment:
So, the field `whitelisted` means, that if it's true, the poll would only be shown to traders in `polls_whitelist`?